web/styles/subscriptions.css

```diff
@@ -688,8 +688,18 @@ h4.user_group_setting_subsection_title {
     }
 }
 
-.group_setting_disabled.pill-container {
+.group_setting_disabled {
     cursor: not-allowed;
```

Comment:
we still need this not-allowed to be here even though we have the rule applied to group-setting-label and pill-container. Otherwise, moving from label to pill container will creater a flicker at the border where the pointer changes to default